modules/administration/pages/maintenance-window.adoc

```diff
@@ -53,8 +53,10 @@ complete procedure, also see above:
 By default, several update channels are configured and enabled for the {susemgr} Server.
```

Comment:
Un related to this PR, shouldn't we use `{productname}` above?

Reply:
probably yes.  We sometimes to drive-by edits ;)

Reply:
done